test/assert-es.js

```diff
@@ -0,0 +1,15 @@
+export default {
+for: Class => {
+    console.log('\x1b[1m', Class.padEnd(60, ' '), '\x1b[0m');
```

Comment:
If this is going to be tested on Web, I'd rather go for:

```js
console.log('%c' + Class, 'font-weight:bold');
```

**and** I would remove the `process.exit(1)` as that would fail either ways.

Reply:
Whoops, yep I'll update this, I pretty much just copied and pasted without really looking at the code.